SPECS-EXTENDED/spirv-tools/spirv-tools.spec

```diff
@@ -45,21 +38,29 @@ Requires:       %{name}-libs%{?_isa} = %{version}-%{release}
 Development files for %{name}
 
 %prep
-%autosetup -p1 -n SPIRV-Tools-%{commit}
+%autosetup -p1 -n SPIRV-Tools-%{version}
 
 %build
 %__mkdir_p %_target_platform
 pushd %_target_platform
+
 %cmake3 -DCMAKE_BUILD_TYPE=Release \
         -DCMAKE_INSTALL_LIBDIR=%{_lib} \
         -DSPIRV-Headers_SOURCE_DIR=%{_prefix} \
         -DPYTHON_EXECUTABLE=%{__python3} \
+        -DSPIRV_TOOLS_BUILD_STATIC=OFF \
         -GNinja ..
-%ninja_build
+%cmake3_build
```

Comment:
Wondering, if we are creating `Ninja` build files, would `%cmake3_build` take care of it ?

Reply:
I'm not an expert on these build tools, unfortunately. But I'm also not entirely clear what is your question. If you're wondering how running `%cmake3_build` ends up building `ninja` stuff, the from the logs it looks like internally `cmake` ends up calling `ninja` at some point. I'm guessing it's doing some additional configuration before/afterwards.